Resources/Prototypes/Entities/Structures/Power/cables.yml

```diff
@@ -184,3 +184,66 @@
         acts: [ "Destruction" ]
   - type: CableVisualizer
     statePrefix: lvcable_
+
+# Explosive cable below
+
+- type: entity
+  id: CableDet
+  placement:
+    mode: SnapgridCenter
+  components:
+  - type: Clickable
+  - type: InteractionOutline
+  - type: Transform
+    anchored: true
+    noRot: true
+  # TODO: Remove both of these, same with CollideOnAnchor
+  - type: Physics
+    bodyType: Static
+    canCollide: false
```

Comment:
Please don't just copy-paste hundreds of lines of yaml, use parenting / composition.

Reply:
Hmmm. I can try to improve it, but the issue is that I'm only using about half the stuff in the CableBase that the other cables use since these cables don't actually interact with the power stuff, so it might not be much better. I'll give it a go.

Reply:
I checked and there's only 6 lines I can recompose that appear in the other cable types. Given that the other cables are pretty different in purpose I think it would just be more confusing.

Reply:
I counted at least 15 shared with cablebase before I stopped counting, you even copy-pasted the comment indicating the below yaml needs removing but still copied it anyway.

Reply:
Weh